examples/st_game/memory/scratch.py

```diff
@@ -0,0 +1,537 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# @Desc   : Scratch类实现（角色信息类）
+
+import datetime
+import json
+import sys
+sys.path.append('../../')
```

Comment:
with `run_ga_game` as entry, no need to add this and not suggest to add `sys.path.append`